specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-01-01/machineLearningServices.json

```diff
@@ -1452,6 +1452,15 @@
           },
           "readOnly": true,
           "description": "The current deployment state of workspace resource. The provisioningState is to indicate states for resource provisioning."
+        },
+        "resourceCmkUri": {
```

Comment:
New properties (even optional ones) should be in a new api-version.  See the ["breaking changes"](https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo) doc.

Reply:
this version is not released yet. We were hoping to combine all jedi changes into the 2020 version

Reply:
Resolving the conversation as per our offline discussion on keeping the same version